examples/digital_credentials_api/signed_request_payload.json

```
@@ -0,0 +1,31 @@
+{
+  "client_id": "client.example.org",
+  "client_id_scheme": "entity_id",
+  "expected_origins": [
+    "https://origin1.example.com",
+    "https://origin2.example.com"
+  ],
+  "response_type": "vp_token",
```

need a response_mode and the jwk being there suggests that the response is expected to be encrypted so w3c_dc_api.jwt 
```suggestion
  "response_type": "vp_token",
  "response_mode": "w3c_dc_api.jwt",
```

---

also response_mode is required per the text in the document so the example should align with that